src/store/modules/order/actions.ts

```diff
@@ -30,6 +41,7 @@ const actions: ActionTree<OrderState, RootState> = {
         const total = resp.data.grouped.orderId.ngroups;
         if (payload.json.params.start && payload.json.params.start > 0) orders = state.list.orders.concat(orders)
         this.dispatch('product/getProductInformation', { orders });
+        this.dispatch('order/fetchStatusChange')
```

Comment:
Why this is needed on every time order is fetch?

Reply:
Sir, it's not needed on orders fetch, I just added it for testing, will remove it.